src/routes/userRoutes.ts

```diff
@@ -1,29 +1,32 @@
 import express from 'express';
 import { handleLogin, handleRegister, handleUpdateUser, handleDeleteUser, handleGetAllUsers, getUserByIdController } from '../controllers/userController';
 import { validateUserLogin, validateUserRegistration } from '../validations/userValidation';
+import { authenticateJWT } from '../middlewares/authMiddleware';
 import { validateRequest } from '../middlewares/validateRequest';
 const router = express.Router();
 
 router.get("/login", (req, res) => {
-    res.render("login");
-  });
+  res.render("login");
+});
+
 router.get("/register", (req, res) => {
-    res.render("register");
-  });
+  res.render("register");
+});
+
+/*router.get('/home', authenticateJWT, (req, res) => {
```

Comment:
why this is commented?

Reply:
This is for EJS. It will not affect the application's functionality, and I will wait until its full completion to uncomment it.